map/framework.cpp

```diff
@@ -732,6 +732,19 @@ void Framework::FillRouteMarkInfo(RouteMarkPoint const & rmp, place_page::Info &
   info.SetIntermediateIndex(rmp.GetIntermediateIndex());
 }
 
+void Framework::FillSpeedCameraMarkInfo(SpeedCameraMark const & speedCameraMark, place_page::Info & info) const
+{
+  info.SetCanEditOrAdd(false);
+  info.SetMercator(speedCameraMark.GetPivot());
+
+  // Primary text is a speed limit, if any.
+  auto const titles = speedCameraMark.GetTitleDecl();
```

Comment:
Are measurement units displayed? They are necessary.

Reply:
I would love to display those but I don't know how to retrieve them

Reply:
@vng knows )